sdk/src/main/scala/com/horizen/AbstractSidechainNodeViewHolder.scala

```diff
@@ -33,6 +31,18 @@ abstract class AbstractSidechainNodeViewHolder[
   override type HIS <: AbstractHistory[TX, H, PMOD, FPI, HSTOR, HIS]
   override type VL <: Wallet[SidechainTypes#SCS, SidechainTypes#SCP, TX, PMOD, VL]
 
+  protected var applyingBlock: Boolean = false
+
+  val maxTxFee: Long = sidechainSettings.wallet.maxTxFee
```

Comment:
Is maxTxFee applicable to Account sidechain too? It is used only in SidechainNodeViewHolder. In addition, for Account sidechain a BigInterger would be more suitable.

Reply:
yes, we will apply it in one of the next tasks (assigned to you :)